week-7/Homework/mandatory/0-thinking-like-a-programmer/task.md

```diff
@@ -18,20 +18,20 @@ The second skill, problem solving, is the most important one. If you get good at
 
 Take a look at the following resources to learn more about problem solving as applied to programming:
 
-[How to THINK like a programmer](https://www.youtube.com/watch?v=NNazO2tMHno)
-[Lessons in problem solving](https://www.freecodecamp.org/news/how-to-think-like-a-programmer-lessons-in-problem-solving-d1d8bf1de7d2/)
-[Computational thinking](https://www.youtube.com/watch?v=qbnTZCj0ugI)
+- [How to THINK like a programmer](https://www.youtube.com/watch?v=NNazO2tMHno)
+- [Lessons in problem solving](https://www.freecodecamp.org/news/how-to-think-like-a-programmer-lessons-in-problem-solving-d1d8bf1de7d2/)
+- [Computational thinking](https://www.youtube.com/watch?v=qbnTZCj0ugI)
 
 After you've watched these videos I'd like you to answer these questions
 
 ## 1. What do you think the most important quality for a programmer is?
 
-<!-- Write your answer here -->
+problem solving
 
 ## 2. When trying to solve a challenge, what should you do first?
 
-<!-- Write your answer here -->
+Understand the issue
 
 ## 3. What should you do if you get stuck?
 
-<!-- Write your answer here -->
+Divide the problem to a sub-problems, start solve them and then connect the points.
```

Comment:
Also, you can look for help on the web or reach a folk out. Also, noticing when we spent to much time on a problem to avoid getting stuck in the rabbit hole for too long.